Replace cluster-wide RBAC resources (#307)
prefapp/charts: charts/argo-workflows-operations/README.md

- `event`: The event to perform on the AKS cluster (`update_image` or `upgrade_version`). `update_image` will update the image of the all the nodes in the cluster, and `upgrade_version` will upgrade the version of the cluster (control plane and nodes).
- `kubernetes_version`: The version of the Kubernetes cluster to upgrade to. This field is required if the `event` is `upgrade_version`.

### Secrets Configuration

To correctly configure the connection to the Azure Key Vault, the following parameters should be defined:

> - `keyVaultName`: The name off the Azure Key Vault 

Reviewer: ```suggestion
- `keyVaultName`: The name of the Azure Key Vault 
```

6. Add the operation-specific logic to the `argo-workflows-operations.rules` template.
7. Add the operation-specific logic to the `argo-workflows-operations.parameters` template.
8. Add the operation-specific logic to the `argo-workflows-operations.workflow` template.

> If you wish to add a new secret to the chasrt, you nedd to follow these steps:

Reviewer: ```suggestion
If you wish to add a new secret to the chart, you need to follow these steps:
```

> 1. Add the Azure Key Vault to the values.yaml file, if it's not already defined.

Reviewer: ```suggestion
1. Add the Azure Key Vault to the `values file, if it's not already defined.
```

> 2. Add the desired configuration for the secret to the values.yaml file.

Reviewer: ```suggestion
2. Add the desired configuration for the secret to the `values file.
```
